src/utils.ts

```diff
@@ -0,0 +1,44 @@
+function Timeout() {
+  const controller = new AbortController();
+  setTimeout(() => controller.abort(), 20);
+  return controller;
+}
+
+export async function checkPort(port: number) {
```

Comment:
This may serve better to check if websocket is open https://github.com/perspect3vism/ad4m-connect/pull/1/files#diff-479ceb5974fc39d426f04ec22e4586e56b0cacf60ccdce7286240942bba6d353R117

Reply:
I do this to just check if the port is occupied or not, if it is I try to connect to executor.